src/Http/OctaneHandler.php

```diff
@@ -29,12 +31,14 @@ public function handleRequest(?HttpRequestEvent $event, ?Context $context): Http
             $response = OctaneClient::handle($request);
         }
 
-        if (! $response->headers->has('Content-Type')) {
-            $response->prepare($request);
-        }
```

Comment:
Figured this should always run, as the `prepare` method does more than setting the `Content-Type` header, and leaves already set stuff to what it is.

Reply:
I added this, because not-so-old Laravel version don't render any 400-500 error templates with a content type.

This was fixed only recently in https://github.com/laravel/framework/pull/43895

Reply:
If I understand you correctly, this was only in there to ensure 404's had a content type.

Since this is now fixed upstream, we should probably remove the call completely.

Users on older versions of Laravel might still encounter the issue, but since the impact seems to be so allow, I personally would be OK with that.

Or alternatively, can we check if the response originated from a NotFoundException?

Reply:
Yeah, hence the check for the `Content-Type` and only call `prepare()` then.

I guess calling `prepare()` on all responses is fine.